gematria/experiments/access_pattern_bm/stl_container_test.cc

```diff
@@ -82,9 +84,11 @@ void BM_STLContainer_Flush(benchmark::State &state) {
   // Create a random container.
   auto container = CreateRandomSTLContainer<Container>(size);
 
-  int sum = 0;
   for (auto _ : state) {
+    int sum = 0;
```

Comment:
Also remove line 100.

Reply:
Something strange seems to be going on there. I had removed the extra `sum = 0` in the previous commit itself, and I don't see that line on [my fork](https://github.com/virajbshah/gematria/blob/bm-measurement-changes/gematria/experiments/access_pattern_bm/stl_container_test.cc). The line is looks like it is still there in the PR diffs, but is shown as removed when I [compare](https://github.com/google/gematria/compare/main...virajbshah:gematria:bm-measurement-changes) google:main with my branch on my fork.

Reply:
Weird. But it looks good here.